reservation/views.py

```diff
@@ -65,21 +69,62 @@ def select_seats(request, movie, schedule):
                 details=schedule_object,
                 seat=seat_obj,
             )
+
             reservations.append(user_reservation)
 
-        send_tickets(request, reservations, schedule_object)
-        return render(request, 'reservation/success.html')
+        confirm_reservation(request, reservations, schedule)
+        return render(request, 'reservation/reservation_email_sent.html')
 
     return render(request, 'reservation/select_seats.html', context)
 
 
-def send_tickets(request, reservation_ids, schedule):
+def success(request, group):
+
+    reservations = list(map(int, group.split('-')))
+
+    for reservation in reservations:
+        Reservation.objects.filter(
+            id=reservation
+        ).update(confirmed=True)
```

Comment:
You could use a lookup to avoid the loop and update all the reservations at once.